src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -694,6 +695,8 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
             currentUserError,
         } = this.state;
 
+        const shouldShowModifyAnnotations = isFeatureEnabled(features, 'activityFeed.modifyAnnotations.enabled');
```

Comment:
Instead of passing it through down to `AnnotationActivityMenu` could we instead use the `withFeatureConsumer` hoc to wrap the `AnnotationActivityMenu`?

Reply:
Yes that sounds good.